Apps/WebClient/src/Server/Controllers/UserProfileController.cs

```diff
@@ -126,8 +126,9 @@ public IActionResult GetUserProfile(string hdid)
             this.logger.LogTrace($"HTTP context user: {JsonConvert.SerializeObject(user, jsonSettings)}");
 
             DateTime jwtAuthTime = GetAuthDateTime(user);
+            string? jwtEmailAddress = user.FindFirstValue(ClaimTypes.Email);
 
-            RequestResult<UserProfileModel> result = this.userProfileService.GetUserProfile(hdid, jwtAuthTime);
+            RequestResult<UserProfileModel> result = this.userProfileService.GetUserProfile(hdid, jwtAuthTime, jwtEmailAddress);
```

Comment:
I am not sure the email from the jwt should be retrieved if it's not the one the user created. Are we thinking about displaying both somewhere?

Reply:
Let's discuss.